Index: 02_Conditional Statements, and Loops/08_how_much_coffee_do_you_need.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/02_Conditional Statements, and Loops/08_how_much_coffee_do_you_need.py b/02_Conditional Statements, and Loops/08_how_much_coffee_do_you_need.py
new file mode 100644
--- /dev/null	(date 1756071071950)
+++ b/02_Conditional Statements, and Loops/08_how_much_coffee_do_you_need.py	(date 1756071071950)
@@ -0,0 +1,34 @@
+tasks_count = 0
+task = input()
+
+while task != 'END':
+
+    first_char = task[0]
+
+
+    if task == 'coding' or task == 'CODING':
+        if first_char.isupper():
+            tasks_count += 2
+        else:
+            tasks_count += 1
+
+    if task == 'dog' or task == 'DOG' or task == 'cat' or task == 'CAT':
+        if first_char.isupper():
+            tasks_count += 2
+        else:
+            tasks_count += 1
+    if task == 'movie' or task == 'MOVIE':
+        if first_char.isupper():
+            tasks_count += 2
+        else:
+            tasks_count += 1
+
+    if tasks_count > 5:
+        print(f"You need extra sleep")
+        break
+    task = input()
+
+if tasks_count <= 5:
+    print(tasks_count)
+
+
Index: 02_Conditional Statements, and Loops/09_sorting_hat.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/02_Conditional Statements, and Loops/09_sorting_hat.py b/02_Conditional Statements, and Loops/09_sorting_hat.py
new file mode 100644
--- /dev/null	(date 1756143802884)
+++ b/02_Conditional Statements, and Loops/09_sorting_hat.py	(date 1756143802884)
@@ -0,0 +1,26 @@
+name = input()
+is_voldemort = False
+
+while name != 'Welcome!':
+    name_length = len(name)
+
+    if name == 'Voldemort':
+        print(f"You must not speak of that name!")
+        is_voldemort = True
+        break
+
+    if name_length < 5:
+        house = f"{name} goes to Gryffindor."
+    elif name_length == 5:
+        house = f"{name} goes to Slytherin."
+    elif name_length == 6:
+        house = f"{name} goes to Ravenclaw."
+    else:
+        house = f"{name} goes to Hufflepuff."
+
+    print(house)
+
+    name = input()
+
+if not is_voldemort:
+    print(f"Welcome to Hogwarts.")
